File: Laborator/Lab 3/JavaClientServer/Networking/src/main/java/dto/ChildDto.java
package dto;

import java.io.Serializable;

public class ChildDto implements Serializable {
    private int id;
    private String name;
    private int age;
    private int IdEvent1;
    private int IdEvent2;

    public ChildDto(int id, String name, int age, int idEvent1, int idEvent2) {
        this.id = id;
        this.name = name;
        this.age = age;
        IdEvent1 = idEvent1;
        IdEvent2 = idEvent2;
    }


    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public int getIdEvent1() {
        return IdEvent1;
    }

    public void setIdEvent1(int idEvent1) {
        IdEvent1 = idEvent1;
    }

    public int getIdEvent2() {
        return IdEvent2;
    }

    public void setIdEvent2(int idEvent2) {
        IdEvent2 = idEvent2;
    }

}

File: Laborator/Lab 3/C# Client-server/Networking/Dto/ChildDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Networking.Dto
{
    [Serializable]
    public class ChildDto
    {
        private int id;
        private string name;
        private int age;
        private int idEvent1;
        private int idEvent2;

        public ChildDto(int id, string name, int age, int idEvent1, int idEvent2)
        {
            this.id = id;
            this.name = name;
            this.age = age;
            this.idEvent1 = idEvent1;
            this.idEvent2 = idEvent2;
        }

        public ChildDto(int id, int age, int idEvent1, int idEvent2)
        {
            this.id = id;
            this.name = "";
            this.age = age;
            this.idEvent1 = idEvent1;
            this.idEvent2 = idEvent2;
        }

        public virtual int ID
        {
            get { return id; }
        }

        public virtual int IdEvent1
        {
            get { return idEvent1; }
        }

        public virtual int IdEvent2
        {
            get { return idEvent2; }
        }

        public virtual int Age
        {
            get { return age; }
        }

        public virtual string Name
        {
            get { return name; }
        }
    }
}
